packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -198,12 +213,20 @@ export default class Tooltip extends PureComponent<TooltipProps> {
         className={cx(
           styles.tooltip,
           getStyle(styles, camelCase(theme)),
-          getStyle(styles, camelCase("padding-size-" + paddingSize)),
           { [styles.withMaxWidth]: withMaxWidth },
           className
         )}
       >
-        {contentValue}
+        {image && <img className={styles.image} src={image} alt="" />}
+        <div className={cx(styles.content, getStyle(styles, camelCase("padding-size-" + paddingSize)))}>
```

Comment:
https://monday.monday.com/boards/3532714909/pulses/6360035918
created a task for it with low priority